Makefile

```diff
@@ -23,29 +23,30 @@ CONTROLLER_GEN_VERSION :=v0.7.0
-$(call add-crd-gen,samples,./samples/v1,./samples/v1,./samples/v1)
-$(call add-crd-gen,security,./security/v1,./security/v1,./security/v1)
-$(call add-crd-gen,securityinternal,./securityinternal/v1,./securityinternal/v1,./securityinternal/v1)
-$(call add-crd-gen,cloudnetwork,./cloudnetwork/v1,./cloudnetwork/v1,./cloudnetwork/v1)
-$(call add-crd-gen,network,./network/v1,./network/v1,./network/v1)
-$(call add-crd-gen,networkoperator,./networkoperator/v1,./networkoperator/v1,./networkoperator/v1)
-$(call add-crd-gen,operatorcontrolplane,./operatorcontrolplane/v1alpha1,./operatorcontrolplane/v1alpha1,./operatorcontrolplane/v1alpha1)
-$(call add-crd-gen,machine-beta,./machine/v1beta1,./machine/v1beta1,./machine/v1beta1)
-$(call add-crd-gen,machine,./machine/v1,./machine/v1,./machine/v1)
-$(call add-crd-gen,monitoring-alpha,./monitoring/v1alpha1,./monitoring/v1alpha1,./monitoring/v1alpha1)
-$(call add-crd-gen,route,./route/v1,./route/v1,./route/v1)
+# $(call add-crd-gen,authorization,./authorization/v1,./authorization/v1,./authorization/v1)
+# $(call add-crd-gen,apiserver,./apiserver/v1,./apiserver/v1,./apiserver/v1)
+# $(call add-crd-gen,config,./config/v1,./config/v1,./config/v1)
+# $(call add-crd-gen,helm,./helm/v1beta1,./helm/v1beta1,./helm/v1beta1)
+# $(call add-crd-gen,console,./console/v1,./console/v1,./console/v1)
+# $(call add-crd-gen,console-alpha,./console/v1alpha1,./console/v1alpha1,./console/v1alpha1)
+# $(call add-crd-gen,imageregistry,./imageregistry/v1,./imageregistry/v1,./imageregistry/v1)
+# $(call add-crd-gen,operator,./operator/v1,./operator/v1,./operator/v1)
+# $(call add-crd-gen,operator-alpha,./operator/v1alpha1,./operator/v1alpha1,./operator/v1alpha1)
+# $(call add-crd-gen,operatoringress,./operatoringress/v1,./operatoringress/v1,./operatoringress/v1)
+# $(call add-crd-gen,sharedresource,./sharedresource/v1alpha1,./sharedresource/v1alpha1,./sharedresource/v1alpha1)
+# $(call add-crd-gen,quota,./quota/v1,./quota/v1,./quota/v1)
+# $(call add-crd-gen,samples,./samples/v1,./samples/v1,./samples/v1)
+# $(call add-crd-gen,security,./security/v1,./security/v1,./security/v1)
+# $(call add-crd-gen,securityinternal,./securityinternal/v1,./securityinternal/v1,./securityinternal/v1)
+# $(call add-crd-gen,cloudnetwork,./cloudnetwork/v1,./cloudnetwork/v1,./cloudnetwork/v1)
+# $(call add-crd-gen,network,./network/v1,./network/v1,./network/v1)
+# $(call add-crd-gen,networkoperator,./networkoperator/v1,./networkoperator/v1,./networkoperator/v1)
+# $(call add-crd-gen,operatorcontrolplane,./operatorcontrolplane/v1alpha1,./operatorcontrolplane/v1alpha1,./operatorcontrolplane/v1alpha1)
+# $(call add-crd-gen,machine-beta,./machine/v1beta1,./machine/v1beta1,./machine/v1beta1)
+# $(call add-crd-gen,machine,./machine/v1,./machine/v1,./machine/v1)
+# $(call add-crd-gen,monitoring-alpha,./monitoring/v1alpha1,./monitoring/v1alpha1,./monitoring/v1alpha1)
+# $(call add-crd-gen,route,./route/v1,./route/v1,./route/v1)
```

Comment:
We'll need to revert this before merging.

Reply:
Yep, just a debugging artifact.